base_layer/core/src/transactions/transaction_components/hex_patch.rs

```diff
@@ -0,0 +1,105 @@
+use std::{fmt, marker::PhantomData};
```

Comment:
The licence text is missing here

Reply:
Moved and fixed here: https://github.com/tari-project/tari_utilities/pull/43